config/tizen/chip-gn/platform/BUILD.gn

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+import("//build_overrides/tizen.gni")
+import("${chip_root}/config/tizen/chip-gn/args.gni")
+
+import("${build_root}/config/linux/pkg_config.gni")
+import("${chip_root}/src/platform/device.gni")
+
+config("tizen_sdk_config") {
+  pkg_config("dlog") {
```

Comment:
pkg_config() shouldn't be inside config() - I'm surprised this is even allowed but I guess since it works the nesting has no effect.

You should be able to remove the config("tizen_sdk_config") as it ends up empty.

Reply:
Done.